lib/Cro/WebApp/Template/Repository.pm6

```diff
@@ -52,7 +56,21 @@ monitor Cro::WebApp::Template::Repository {
     }
 }
 
-my $template-repo = Cro::WebApp::Template::Repository.new;
+monitor Cro::WebApp::Template::ReloadingRepository is Cro::WebApp::Template::Repository {
+    has %!abs-path-to-mtime;
+
+    method resolve-absolute($abs-path --> Promise) {
+        my $modified = $abs-path.IO.modified;
+        if (%!abs-path-to-mtime{$abs-path} // 0) != $modified {
+            self.refresh($abs-path)
+        }
+        %!abs-path-to-mtime{$abs-path} = $modified;
+        callsame
+    }
+
+}
+
+my $template-repo = %*ENV<CRO_DEBUG> ?? Cro::WebApp::Template::ReloadingRepository.new !! Cro::WebApp::Template::Repository.new;
```

Comment:
maybe `CRO_DEBUG` should be stored somewhere else

Reply:
Maybe, will have to think about that (though it's not really a blocker). I would rather it was called `CRO_DEV` though; it's a feature aimed at development time in general, not just for debugging.